package.json

```diff
@@ -23,6 +23,6 @@
   },
   "devDependencies": {
     "ava": "^0.18.2",
-    "webpack": "^2.2.1"
+    "webpack": "2.2.1"
```

Comment:
if it isn't changed, it breaks unless you install with yarn unless the absolute paths are fixed